app/src/main/java/net/squanchy/schedule/view/TalkEventItemView.kt

```diff
@@ -0,0 +1,61 @@
+package net.squanchy.schedule.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.TextView
+import net.squanchy.R
+import net.squanchy.eventdetails.widget.ExperienceLevelView
+import net.squanchy.schedule.domain.view.Event
+import net.squanchy.support.lang.Optional
+import net.squanchy.support.widget.SpeakerView
+import org.joda.time.format.DateTimeFormat
+
+class TalkEventItemView @JvmOverloads constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int = R.attr.cardViewDefaultStyle) :
```

Comment:
layout styling

Reply:
What do you mean?

Reply:
```
class TalkEventItemView @JvmOverloads constructor(
  context: Context, 
  attrs: AttributeSet, 
  defStyleAttr: Int = R.attr.cardViewDefaultStyle
) : ...
```

Reply:
Dat 👍

Reply:
0e9919d96f06b9978776c48bfd82bccfc12c21d3